### PR TITLE
feat: add emissions to devs chart

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
https://olas-website-git-tanya-emissions-to-devs-autonolas.vercel.app/olas-token

image

### CODE REVIEW

File: common-util/graphql/queries.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { gql } from "graphql-request";
+
+export const emissionsQuery = gql`
+  {
+    epoches(orderBy: startBlock) {
```
Comment (Tanya-atatakai) [inline common-util/graphql/queries.js:5]: this plural was automatically generated, unfortunately I'm unable to provide the correct one

File: components/OlasTokenPage/EmissionsToDevs.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                    gridLines: {
+                      color: 'white',
+                    },
+                  },
+                  y: {
+                    title: {
+                      display: true,
+                      text: 'OLAS Emitted',
+                    },
+                    max: Math.max(...chartPoints) * 1.3,
```
Comment (truemiller) [inline components/OlasTokenPage/EmissionsToDevs.jsx:83]: i guess the `1.3` is to add some margin above? unsure. 
can either make it a constant or add comment on it's purpose

**Review by truemiller — DISMISSED** [review]
looks really clean!
one tiny thing, but not really an issue

File: components/OlasTokenPage/EmissionsToDevs.jsx (hunk 2 above)
Comment (Tanya-atatakai) [reply components/OlasTokenPage/EmissionsToDevs.jsx:83]: you're correct, yes. makes sense, will do

**Review by truemiller — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

**Pull request merged**